src/coreclr/jit/gentree.cpp

```diff
@@ -18927,6 +18943,13 @@ bool GenTree::isContainableHWIntrinsic() const
             return true;
         }
 
+        case NI_Vector128_get_Zero:
+        case NI_Vector256_get_Zero:
+        {
+            // These HWIntrinsic operations are contained as part of Sse41.Insert
```

Comment:
I assume it's also true for Vector256? (Avx Insert). just for my understanding

Reply:
There is no `Insert` for `Avx/Avx2`, only `InsertVector128`.